pyproject.toml

```
@@ -118,3 +140,43 @@ artifacts = ["/yt_dlp/extractor/lazy_extractors.py"]
 [tool.hatch.version]
 path = "yt_dlp/version.py"
 pattern = "_pkg_version = '(?P<version>[^']+)'"
+
+[tool.ruff]
+required-version = "0.2.1"
+line-length = 120
+
+[tool.ruff.lint]
+ignore = [
+    "E402", # module level import not at top of file
+    "E501", # line too long
+    "E731", # do not assign a lambda expression, use a def
+    "E741", # ambiguous variable name
+]
+select = [
+    "E", # pycodestyle errors
+    "W", # pycodestyle warnings
+    "F", # pyflakes
+    "I", # import order
```

from the notes of grub
```suggestion
    # "I", # import order
```